### PR TITLE
Fix splitter

### PR DESCRIPTION
**Pull request by mrdrogdrog**
### Component/Part
Split component

### Description
This PR fixes the split component. If you stop holding down your mouse button while the mouse is not on the document after starting the resizing mode, then the splitter isn't stuck in resizing mode anymore.
You can also move your mouse over the whole document to resize the split.

### Steps

- [x] Added implementation
- [x] Added / updated documentation
- [x] I read the [contribution documentation](https://github.com/hedgedoc/react-client/blob/main/CONTRIBUTING.md) and signed-off my commits to accept the DCO.

### CODE REVIEW

**Review by ErikMichelson — CHANGES_REQUESTED** [review]
When dragging the splitter and you're leaving and re-entering the document area, the splitter extends only in one direction.


https://user-images.githubusercontent.com/52606896/121597670-47219400-ca41-11eb-8bc8-ffae781002a6.mp4

File: src/components/editor-page/editor-page.tsx [hunk 1]
```diff
@@ -112,40 +112,40 @@ export const EditorPage: React.FC = () => {
...
-        onTaskCheckedChange={ SetCheckboxInMarkdownContent }
-        onFrontmatterChange={ setNoteFrontmatter }
-        onScroll={ onMarkdownRendererScroll }
-        scrollState={ scrollState.rendererScrollState }
-        rendererType={ RendererType.DOCUMENT }
+        frameClasses={'h-100 w-100'}
+        markdownContent={markdownContent}
+        onMakeScrollSource={setRendererToScrollSource}
+        onFirstHeadingChange={updateNoteTitleByFirstHeading}
+        onTaskCheckedChange={SetCheckboxInMarkdownContent}
```
[the remark this answers is not in the document]
Comment (mrdrogdrog) [reply src/components/editor-page/editor-page.tsx]: Don't know. I fixed it.

**Comment by mrdrogdrog** [comment]
@ErikMichelson I also fixed the resizing issue

**Review by ErikMichelson — APPROVED** [review]

**Review by DerMolly — APPROVED** [review]
LGM

**Pull request merged**